src/pages/MyTokens/Filters/StatusFilter.tsx

```diff
@@ -12,66 +13,127 @@ interface StatusFilterProps {
+
+  const hideFixedPrice = useMemo(() => (timedAuction), [timedAuction]);
+  const hideTimedAuction = useMemo(() => (fixedPrice), [fixedPrice]);
+  const hideOnSale = useMemo(() => (notOnSale), [notOnSale]);
+  const hideNotOnSale = useMemo(() => (fixedPrice || timedAuction || onSell), [fixedPrice, timedAuction, onSell]);
+
   return (
-    <Accordion title={'Status'}
-      isOpen={true}
-      onClear={onClear}
-      isClearShow={onSell || fixedPrice || timedAuction || notOnSale}
-      testid={`${testid}-accordion`}
-    >
-      <StatusFilterWrapper>
-        <Checkbox
-          checked={!!onSell}
-          label={'My NFTs on sale'}
-          size={'m'}
-          onChange={onMyNFTsOnSellChange}
-          testid={`${testid}-myNft-checkbox`}
-        />
-        <Checkbox
-          checked={!!fixedPrice}
-          label={'Fixed price'}
-          size={'m'}
-          onChange={onFixedPriceChange}
-          testid={`${testid}-fixedPrice-checkbox`}
-        />
-        <Checkbox
-          checked={!!timedAuction}
-          label={'Timed auction'}
-          size={'m'}
-          onChange={onTimedAuctionChange}
-          testid={`${testid}-timedAuction-checkbox`}
-        />
-        <Checkbox
-          checked={!!notOnSale}
-          label={'Not on sale'}
-          size={'m'}
-          onChange={onNotOnSaleChange}
-          testid={`${testid}-notOnSale-checkbox`}
-        />
-      </StatusFilterWrapper>
-    </Accordion>
+    <>
+      <Accordion
+        title={'Selling method'}
+        isOpen={true}
+        onClear={onMethodClear}
+        testid={`${testid}-selling-method-accordion`}
+      >
+        <StatusFilterWrapper>
+          {!hideFixedPrice && <Checkbox
+            checked={!!fixedPrice}
+            label={'Fixed price'}
+            size={'m'}
+            onChange={onFixedPriceChange}
+            testid={`${testid}-fixedPrice-checkbox`}
+          />}
+          {!hideTimedAuction && <Checkbox
+            checked={!!timedAuction}
+            label={'Timed auction'}
+            size={'m'}
+            onChange={onTimedAuctionChange}
+            testid={`${testid}-timedAuction-checkbox`}
+          />}
+        </StatusFilterWrapper>
+      </Accordion>
+      {selectedAccount && <Accordion
+        title={'Status'}
+        isOpen={true}
+        onClear={onStatusClear}
+        testid={`${testid}-status-accordion`}
+      >
+        <StatusFilterWrapper>
+          {!hideOnSale && <Checkbox
+            checked={!!onSell}
+            label={'On sale'}
+            size={'m'}
+            onChange={onMyNFTsOnSellChange}
+            testid={`${testid}-myNft-checkbox`}
+          />}
+          {!hideNotOnSale && <Checkbox
+            checked={!!notOnSale}
+            label={'Not on sale'}
+            size={'m'}
+            onChange={onNotOnSaleChange}
+            testid={`${testid}-myBids-checkbox`}
+          />}
+        </StatusFilterWrapper>
+      </Accordion>}
+    </>
   );
+
+  // return (
```

Comment:
delete code, we can always find it in git history

Reply:
done